addons/hr/models/res_partner.py

```diff
@@ -32,10 +32,19 @@ def _compute_employees_count(self):
 
     def action_open_employees(self):
         self.ensure_one()
+        if len(self.employee_ids) > 1:
+            return {
+                'name': _('Related Employees'),
+                'type': 'ir.actions.act_window',
+                'res_model': 'hr.employee',
+                'view_mode': 'kanban,tree,form',
+                'domain': [('id', 'in', self.employee_ids.ids)],
+            }
         return {
-            'name': _('Related Employees'),
+            'name': _('Employee'),
             'type': 'ir.actions.act_window',
             'res_model': 'hr.employee',
-            'view_mode': 'kanban,tree,form',
-            'domain': [('id', 'in', self.employee_ids.ids)],
+            'res_id': self.employee_ids.ids,
```

Comment:
```suggestion
            'res_id': self.employee_ids.id,
```

Reply:
What about this one? :thinking:

Reply:
also it will not hurt if we put `'view_mode': 'form',` as well :)

Reply:
changed it and it works but idk why ? This action is supposed to land in Kanban view because there are multiple records, how does it go to the kanban view if I specify form view mode ?